### PR TITLE
TheNextWeb-clone

### PR DESCRIPTION
**Pull request by FrederickMih**
We have successfully structured out the next web clone page, including the mobile, tablet, and desktop versions in addition with media queries. As a result of this, we need some feedbacks in order to improve the look of our page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,4 +70,4 @@ Give a ⭐️ if you like this project!
 
 ## 📝 License
 
-This project is [MIT](lic.url) licensed.
+This project is [MIT](lic.url) licensed.
```
Comment (arslanbisharat) [inline README.md]: - [ ] **Wrong Path**: Please add the path to the License. You don't have any URL right now.

File: README.md [hunk 2]
```diff
@@ -1,22 +1,19 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# TheNextWeb-clone
 
-> One paragraph statement about the project.
+> This project is the clone of the next web page
```
Comment (arslanbisharat) [inline README.md]: - [ ] **More Information:** Please add more details about your project here. Please tell us why you created this project and how you did it.

File: README.md [hunk 3]
```diff
@@ -1,22 +1,19 @@
...
 
-![screenshot](./app_screenshot.png)
+![screenshot](./assets/images/thenextweb.png)
```
Comment (arslanbisharat) [inline README.md]: - [ ] [Optional] You can add screenshots from different screen-sizes to showcase your project effectively.

File: header.css [hunk 4]
```diff
@@ -0,0 +1,79 @@
+@import url('https://fonts.googleapis.com/css2?family=Play:wght@400;700&display=swap');
+
+
+.header-tablet {
+  /* border: 2px solid red; */
```
Comment (arslanbisharat) [inline header.css]: - [ ] **Commented Code:** Please remove commented code from your project. It is not a good practice to write commented code in your project. Please keep it clean as much as possible.

File: header.css [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+  margin-top: 0;
+  /* position: relative; */
```
Comment (arslanbisharat) [inline header.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: header.css [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+}
+
+.header-tablet .logo {
+  border: 2px solid blue;
+  background-color: #000;
+  display: inline;
+  padding: 1em;
+  /* position: absolute; */
```
Comment (arslanbisharat) [inline header.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: header.css [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+}
+
+.header-tablet .logo img {
+    width: 8%;
+}
+
+/* ----------TOP NAV BAR------------ */
+
+.nav-top {
+  /* border: 2px solid red; */
```
Comment (arslanbisharat) [inline header.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: header.css [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+  list-style-type: none;
+  padding-top: 0.3em;
+  padding-bottom: 0.3em;
+  padding-left: 0.3em;
+  margin: 0;
+}
+
+.nav-top ul li a,
+.nav-bottom ul li a {
+  /* border: 1px solid red; */
```
Comment (arslanbisharat) [inline header.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: header.css [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+}
+
+.about {
+    margin-left: auto;
+}
+
+/* -------------BOTTOM NAV BAR------------- */
+
+.nav-bottom {
+    /* border: 2px solid red; */
```
Comment (arslanbisharat) [inline header.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: tablet.css [hunk 10]
```diff
@@ -0,0 +1,124 @@
...
+        padding-top: 40%;
+    }
+
+    .world h2 {
+        font-size: 2rem;
+    }
+
+    .gear {
+        grid-area: gear;
+        /* border: 2px solid red; */
```
Comment (arslanbisharat) [inline tablet.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: tablet.css [hunk 11]
```diff
@@ -0,0 +1,124 @@
...
+        margin-bottom: 0;
+    }
+
+    .distract {
+        grid-area: distract;
+        /* border: 2px solid red; */
```
Comment (arslanbisharat) [inline tablet.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: tablet.css [hunk 12]
```diff
@@ -0,0 +1,124 @@
...
+        margin-top: 0;
+    }
+
+    /* ---------NEWS GRID-------- */
+
+    .grid-news-content  {
+        display: grid;
+        grid-template-columns: repeat(3, 1fr);
+        /* grid-template-rows: repear(3, 1fr); */
```
Comment (arslanbisharat) [inline tablet.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: index.html [hunk 13]
```diff
@@ -0,0 +1,675 @@
...
+            <div class="header-mobile">
+                <ul>
+                    <li><a class="TNW-logo" href="#"><img src="assets/icons/tnw.svg" alt="TNW Logo"></a></li>
+                    <li class="search-icon"><a href="#"><i class="fa fa-search" aria-hidden="true"></i></a></li>
+                    <li><a href="#"><i class="fa fa-bars" aria-hidden="true"></i></a></li>
+                </ul>
+            </div>
+
+            <div class="header-tablet">
+                <!-- <div class="logo"><img src="assets/icons/tnw.svg" alt="TNW logo"></div> -->
```
Comment (arslanbisharat) [inline index.html]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 14]
```diff
@@ -0,0 +1,601 @@
...
+}
+
+ul li + li {
+  margin-top: 0;
+}
+
+/* ----------------CSS for HEADER--------------- */
+
+header {
+  /* border: 5px solid green; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 15]
```diff
@@ -0,0 +1,601 @@
...
+}
+
+.header-mobile {
+  /* border: 3px solid blue; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 16]
```diff
@@ -0,0 +1,601 @@
...
+  box-shadow: 0 5px #f1f1f1;
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+  background-color: #fff;
+}
+
+.header-mobile ul {
+  /* border: 1px solid red; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 17]
```diff
@@ -0,0 +1,601 @@
...
+  justify-content: space-between;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+
+}
+
+.header-mobile ul .TNW-logo {
+  /* background-color: #000; */
+  /* border: 2px solid red;   */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 18]
```diff
@@ -0,0 +1,601 @@
...
+}
+
+.header-mobile ul .TNW-logo img {
+  width: 24%;
+  height: auto;
+  background-color: #000;
+  /* border: 2px solid red; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 19]
```diff
@@ -0,0 +1,601 @@
...
+}
+
+.header-desktop {
+  display: none;
+}
+
+/* ----------CSS for main content-----------  */
+
+.content{
+  /* border: 2px solid red; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 20]
```diff
@@ -0,0 +1,601 @@
...
+  margin-top: 0;
+  padding: 0.5em;
+  /* background-color: yellow; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 21]
```diff
@@ -0,0 +1,601 @@
...
+}
+
+/* ------------content-cover section------------ */
+
+.content-cover {
+  /* border: 2px solid red; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 22]
```diff
@@ -0,0 +1,601 @@
...
+}
+
+.cover {
+  /* border: 2px solid blue; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 23]
```diff
@@ -0,0 +1,601 @@
...
+  color: #fff;
+}
+
+.cover h4,
+.exclusive-feature .text h4 {
+  font-family: var(--square-font);
+  font-weight: 400;
+  margin-bottom: 0;
+  display: inline-block;
+  /* border: 2px solid red; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 24]
```diff
@@ -0,0 +1,601 @@
...
+  text-transform: uppercase;
+}
+
+.cover h2 {
+  font-size: 1.4rem;
+  font-weight: bolder;
+  margin-top: 0;
+  margin-bottom: 0;
+  cursor: pointer;
+  /* border: 2px solid green; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 25]
```diff
@@ -0,0 +1,601 @@
...
+}
+
+.cover p {
+  font-size: 0.8rem;
+  color: #f1f1f1;
+  /* border: 1px solid red; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 26]
```diff
@@ -0,0 +1,601 @@
...
+
+.distract {
+  background: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 1)), url("assets/images/electric-car.jpg");
+}
+
+
+/* ---------Grid news section-------- */
+
+.grid-news {
+  /* border: 5px solid green; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 27]
```diff
@@ -0,0 +1,601 @@
...
+  padding-left: 0.5em;
+  padding-right: 0.5em;
+}
+
+.grid-news-title {
+  display: flex;
+  justify-content: space-between;
+  /* border: 1px solid blue; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 28]
```diff
@@ -0,0 +1,601 @@
...
+}
+
+.grid-news-title a + a {
+  margin-top: 0;
+}
+
+.grid-news-title a {
+  color: red;
+  text-decoration: none;
+  /* border: 1px solid blue; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 29]
```diff
@@ -0,0 +1,601 @@
...
+
+.focus-less {
+  font-size: 0.9rem;
+  align-self: flex-end;
+}
+
+/* -----------grid-news-contents----------- */
+
+.grid-news-content {
+  /* border: 2px solid red; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 30]
```diff
@@ -0,0 +1,601 @@
...
+
+.news-item {
+  /* border: 2px solid green; */
+  display: flex;
+}
+
+.news-item img {
+ height: 80px;
+ width: 80px;
+ /* border: 2px solid red; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 31]
```diff
@@ -0,0 +1,601 @@
...
+ align-self: center;
+}
+
+.news-item img:hover {
+  cursor: pointer;
+}
+
+.news-item-desc {
+  /* border: 2px solid red; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 32]
```diff
@@ -0,0 +1,601 @@
...
+}
+
+.source > a:hover {
+  color: #000;
+}
+
+/* CSS for latest section */
+
+.latest {
+  /* border: 2px solid red; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 33]
```diff
@@ -0,0 +1,601 @@
...
+  padding-left: 0.5em;
+  padding-right: 0.5em;
+}
+
+.latest > header {
+  /* border: 2px solid blue; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 34]
```diff
@@ -0,0 +1,601 @@
...
+  color: red;
+}
+
+.latest header h2 a:hover,
+.exclusive-tile h2 a:hover {
+  color: black;
+}
+
+.latest header ul {
+  /* border: 2px solid blue; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 35]
```diff
@@ -0,0 +1,601 @@
...
+  color: red;
+  text-decoration: none;
+}
+
+.latest header ul li a:hover {
+  color: black;
+} 
+
+.latest-content {
+  /* border: 2px solid red; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 36]
```diff
@@ -0,0 +1,601 @@
...
+}
+
+.latest-tile {
+  /* border: 2px solid green; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 37]
```diff
@@ -0,0 +1,601 @@
...
+.tile-head {
+  position: relative;
+  margin-bottom: 1.5em;
+  margin-top: 2em;
+}
+
+.tile-head img {
+  height: 60px;
+  width: 60px;
+  /* border: 2px solid red; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 38]
```diff
@@ -0,0 +1,601 @@
...
+  position: absolute;
+  top: 0.5em;
+  left: 0.5em;
+}
+
+.tile-head-title {
+  background-color: #F34A24;
+  /* border: 2px solid green; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 39]
```diff
@@ -0,0 +1,601 @@
...
+  color: #fff;
+  text-decoration: none;
+  padding: 0.5em 0;
+  margin-left: 80px;
+  font-size: 1.2rem;
+  font-weight: 700;
+}
+
+.rate {
+  /* border: 2px solid blue; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 40]
```diff
@@ -0,0 +1,601 @@
...
+  font-size: 0.9rem;
+  font-weight: 400;
+  font-family: var(--square-font);
+}
+
+.tile-target-link {
+  font-size: 0.85rem;
+  padding-left: 0.5em;
+  color: var(--text-color-light);
+  /* border: 2px solid blueviolet; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 41]
```diff
@@ -0,0 +1,601 @@
...
+
+/* -------------EXCLUSIVE SECTION---------- */
+
+.exclusive {
+  padding-left: 0.5em;
+  padding-right: 0.5em;
+}
+
+.exclusive-tile {
+  /* border: 2px solid blueviolet; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 42]
```diff
@@ -0,0 +1,601 @@
...
+.exclusive-tile h2 a {
+  text-decoration: none;
+}
+.popular {
+  background-color: #000;
+  margin-bottom: 0;
+}
+
+.exclusive-feature {
+  /* border: 2px solid red; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 43]
```diff
@@ -0,0 +1,601 @@
...
+  justify-content: space-between;
+  color: red;
+  background-color: #FDE8E4;
+  margin-top: 0;
+}
+
+.exclusive-feature img {
+  height: 100px;
+  width: 100px;
+  /* border: 3px solid green; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 44]
```diff
@@ -0,0 +1,601 @@
...
+}
+
+.exclusive-feature > a {
+  margin-top: 0;
+}
+
+.exclusive-feature .text {
+  /* border: 1px solid blue; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 45]
```diff
@@ -0,0 +1,601 @@
...
+  padding: 0 0.8em;
+}
+.exclusive-feature .text h4 {
+  cursor: auto;
+  font-size: 0.9rem;
+}
+
+.exclusive-feature .text p {
+  /* border: 1px solid red; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 46]
```diff
@@ -0,0 +1,601 @@
...
+  text-decoration: none;
+  font-weight: 700;
+}
+
+.exclusive-feature .text p a:hover{
+  text-decoration: underline;
+}
+
+.exclusive-links {
+  /* border: 2px solid green; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 47]
```diff
@@ -0,0 +1,601 @@
...
+.non-strike {
+  color: #6c6c6c;
+  font-size: 0.9rem;
+}
+
+
+/* ------------FOOTER SECTION----------- */
+
+.footer {
+  /* border: 2px solid red; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 48]
```diff
@@ -0,0 +1,601 @@
...
+}
+
+.footer-top {
+  /* border: 2px solid blue; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 49]
```diff
@@ -0,0 +1,601 @@
...
+  padding-top: 2em;
+  padding-bottom: 2em;
+}
+
+.footer-top .icons ul,
+.top-footer-menu ul {
+  list-style-type: none;
+  padding: 0;
+  display: flex;
+  /* border: 1px solid green; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 50]
```diff
@@ -0,0 +1,601 @@
...
+}
+
+.top-footer-menu ul li a:hover {
+  color: #fff;
+}
+
+.footer-bottom {
+  background-color: #000;
+  margin-top: 0;
+  /* border: 1px solid red; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 51]
```diff
@@ -0,0 +1,601 @@
...
+  color: #fff;
+  padding-top: 1.5em;
+  padding-bottom: 1.5em;
+  text-align: center;
+}
+
+.footer-bottom p {
+  /* border: 1px solid red; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

File: style.css [hunk 52]
```diff
@@ -0,0 +1,601 @@
...
+  font-size: 0.8rem;
+  color: #787878;
+}
+
+.footer-bottom p > span {
+  display: block;
+  /* border: 1px solid blue; */
```
Comment (arslanbisharat) [inline style.css]: - [ ] **Commented Code:** Please remove commented code from your project.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Request changes 😳

I am very happy to see this project and I can see you worked hard on this project. There are only a few issues that we need to address and I am sure that you will end up with a perfect project

_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_

This is a great opportunity to learn. Microverse is here to help you. 

**Please have a look at the suggestions to get more understanding.**

- [ ]  **Commented Code:** Please remove commented code from your project. It is not a good practice to write commented code in your project. Please keep it clean as much as possible. 

- [ ] Please use semantic HTML. You are missing the **main** and **footer** tags. 
- [ ] [Optional] Please add your CSS files into a single folder and then add your assets and CSS folder in one single folder. This will help you to make your project clean. 

- [ ] **Linter**: Please remove Linter errors.

- [ ] You have too many empty rules in CSS files. Please remove them. 
### Desktop
- [ ] You have unnecessary white space before the navbar. Please remove that white space. 
chrome-capture (9)

- [ ] You are missing the TNW logo on the Desktop version from the left big image. Please work on it. 
chrome-capture (10)

- [ ] Please move the text and other content down as present on the original page. 
chrome-capture (11)

- [ ] Please move the items from left to the right side opposite to the` Latest funding rounds on Index.co
`
chrome-capture (12)

- [ ] Please make the navbar sticky on scrolling as present on the original page. 

- [ ] Please correct the path of the images. The images are not rendering because of the wrong path.  You must add ./ before a path declaration. 

chrome-capture (13)

- [ ] Please add different content (headings and other text including images) for the whole page. This will make your project outstanding. 

### Medium

- [ ] TWN logo appears at the leftmost part on navbar but that part is missing in your clone. Please work on it. 

chrome-capture (14)

### SMALL

- [ ] The TNW logo on small doesn't match the one present on the original page. Please work on it. 


**Please make the changes and request a re-review for the given project**

If you get stuck somewhere, don't hesitate to reach me out or schedule s 1:1 meeting with a Full-time TSE. 

Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

- [ ] Customize readme with information specific to your project. The Getting started section has no relevant information about how to get started with this specific proyect.
image

- [ ] The live demo link is not working on the README. Link is broken.

- [ ] Images are not displaying.  Please correct the path of the images. The images are not rendering because of the wrong path. You must add `./` before a path declaration on the HTML and CSS files.
image
image

- [ ] Tablet view should go from 768px to 426px. 

- [ ]  The TNW logo in the 3 image section is still not displaying

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Comment by descholar-ceo** [comment]
# This Pull Request is under review by [descholar-ceo](https://github.com/descholar-ceo) :eyes: :eyes: 
## Please wait a bit ...

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @KumarAmitt and @FrederickMih  :handshake: :handshake: :handshake: 

:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
#### :warning: However we need to make few changes in your readme file to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [ ] The license link from your readme file doesn't exists
- [ ] The issues link you provided in your readme file doesn't work, it should be fixed before requesting another review
- [ ] The linkedin profile link of Amit Kumar provided in your readme file doesn't work, it should be fixed before requesting another review
- [ ] In the `Getting started section` from your readme file there are some steps which are not clear users who are not already familiar with git/github, read below text and improve it by answering the questions I put:`

```bash
Setup
See the product documentation for installation and get running. => This sentence is not clear, which product should a user read its docs, and where to find those docs? My Suggestion: I don't see any point to put this sentence, you can just remove it.

Usage
Clone the folder on your local machine. => How should a user clone that folder? Is it really a folder that I should clone? (Answer: It's a repository instead), so then specify which command a user should run to clone this repository.

```

# [Optional changes] :yellow_circle: 

- [ ] The top three articles are taking a big height, please reduce their height and so that they will look exactly like the original version

:warning: Your commit messages should be consistently in imperative mode or present tense, _i.e.:_ you should have written `Fix CSS linter errors` instead of writing `Fixing CSS linter errors`, refer [to this article for more details on how to write a good && professional commit message](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

___
_If anything seems to be unclear to you, reachout and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

**Comment by mimipeshy** [comment]
## STATUS: APPROVED 🥳  🍾 🕺  

Hey Team @KumarAmitt and  @FrederickMih 👋 👋 , congratulations on working so hard on this project. Good job on working on all the feedback given by all TSE's. You are on your way to becoming a world-class developer 👩‍💻 . Go ahead and merge.
All the best in future projects.

Regards
Peris Ndanu

**Pull request merged**